docs/bridging/bridging-fa.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Bridging FA tokens between Tezos layer 1 and Etherlink
+sidebar_label: Bridging FA tokens
+---
+
+You can bridge FA standards-compliant tokens in and out of Etherlink by deploying contracts that move the tokens.
+You can call those contracts to bridge the tokens or request that the Etherlink team add them to the bridge UI.
+
+The Tezos FA standards are token standards like the Ethereum ERC-20, ERC-721, and ERC-1155 standards.
+For more information about the FA standards, see [Token standards](https://docs.tezos.com/architecture/tokens#token-standards) on docs.tezos.com.
+
+Two bridging operations are available:
+
+- Bridging tokens from Tezos layer 1 to Etherlink is referred to as _depositing_ tokens.
+Depositing an FA token locks it in a contract on layer 1 and mints an equivalent ERC-20 token on Etherlink.
+- Bridging tokens from Etherlink to Tezos layer 1 is referred to as _withdrawing_ tokens.
+Withdrawing an FA token burns the ERC-20 token on Etherlink and unlocks the equivalent token on Tezos layer 1.
+
+:::note
+<h3>Bridging time</h3>
+Tokens that you bridge from Tezos layer 1 to Etherlink are available for use on Etherlink immediately.
+
+Tokens that you bridge from Etherlink to Tezos layer 1 are available for use on Tezos in 15 days.
+
+This delay is caused by the Smart Rollup refutation period.
+As with all Smart Rollups, Etherlink nodes post commitments about their state to Tezos layer 1, including incoming bridging transactions, on a regular schedule.
+Other nodes have the length of the refutation period (14 days) to challenge those commitments.
+At the end of the refutation period, the correct commitment is cemented, or made final and unchangeable.
```

Comment:
Nit but technically it doesn't have to be the correct one as long as there are no validators challenging an incorrect committment

Reply:
I'd rather leave this as "correct" for marketing purposes, instead of going into the "optimistic" part of SCORUs here. This part of the refutation process (that if there is only one commitment, it is not challenged) is covered in the [Smart Rollup](https://docs.tezos.com/architecture/smart-rollups#refutation-periods) docs.

Reply:
Okay, sure.